flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
I don't have any AArch64 devices to test the app

Reply:
And this game is Desktop-only

Reply:
neither is a valid excuse. There are aarch64 desktops. And it surely can't be tested if it is not built.

Reply:
So I should just delete this file to see what happens?

Reply:
Yes

Reply:
Ok, thanks